frontend/spec/features/representative_file_version_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'spec_helper'
 require 'rails_helper'
 
-describe 'Representative File Version', js: true do
+describe 'Representative File Version', js: true, skip: true do
```

Comment:
If we are trying to investigate whether the test failures are related to Java 8, do we really want to skip these tests?

Reply:
@donaldjosephsmith good point! I thought I will disable them in this PR as a temporary measure so that we can continue working and I will work on a separate PR on https://archivesspace.atlassian.net/browse/ANW-1893 to reenable and fix these specs.

Reply:
Ok, makes sense.